src/http/server/handlers/query_input.rs

```diff
@@ -34,9 +33,9 @@ async fn handler(
     let permit = transport_sender.reserve().await?;
 
     let input_stream = RequestParts::new(req)
-        .extract::<ByteArrStream>()
-        .await
-        .map_err(|_| Error::BadQueryString("TODO: move ByteArrStream to the right package".into()))?
+        .extract::<ByteArrStreamFromReq>()
+        .await?
+        .0
         .and_then(|bytes| futures::future::ok(bytes.to_vec()))
         .map_err(TransportError::from);
     let query_input = QueryInput {
```

Comment:
I think we need another todo here. `FieldType` should not be sent twice per query. RC initiates the query telling us what field type to use and then submits the input assuming we know the field type already

Reply:
right. this is one of the points i wanted to discuss over VC. i have a proposed solution and wanted to pass it by you

Reply:
this field has been removed in the last PR: https://github.com/thurstonsand/raw-ipa/pull/12 (see commit: https://github.com/thurstonsand/raw-ipa/pull/12/commits/dd1fac7d5fe15ca8d00373b45193fb656226c779)